release/release_test.go

```diff
@@ -50,3 +52,42 @@ func (s *ReleaseTestSuite) TestOverride(c *C) {
 	c.Check(release.String(), Equals, "10.06-personal")
 	c.Check(release.Get(), DeepEquals, rel)
 }
+
+func makeMockLsbRelease(c *C) string {
+	// FIXME: use AddCleanup here once available so that we
+	//        can do release.SetLsbReleasePath() here directly
```

Comment:
Ah, interesting. That indeed makes me wish for something along the lines of addCleanup, as it's inside a helper which may be called or not, and the logic remains completely independent from the suite. I like it, thanks.